CI reports:
```
2022-11-01T11:50:17.6170914Z === RUN   TestHTTPListenerAddr
2022-11-01T11:50:17.6171167Z windows run 'netsh int ipv4 set dynamicport tcp'  Error:  exit status 1
2022-11-01T11:50:17.6171362Z --- FAIL: TestHTTPListenerAddr (1.59s)
```

--- a/.github/workflows/go-lint.yml
+++ b/.github/workflows/go-lint.yml
@@ -34,6 +34,7 @@ jobs:
           CGO_ENABLED: 0
           GO111MODULE: on
         run: |
+          netsh int ipv4 set dynamicport tcp start=60000 num=61000
           go build --ldflags="-s -w" -o %GOPATH%\bin\minio.exe
           go test -v --timeout 50m ./...
       - name: Build on ${{ matrix.os }}

